src/app/shared/enums/events.ts

```diff
@@ -47,3 +47,15 @@ export enum NotificationEventType {
   RESCHEDULED = 'notification_rescheduled',
   TEST = 'notification_test'
 }
+
+export enum KeyboardEventType {
+  FOCUS = 'focus',
+  BLUR = 'blur',
+  ENTER = 'enter'
+}
+
+export enum NextButtonEventType {
+  AUTO = 'auto',
```

Comment:
Does auto mean that they will be automatically moved to next question when they select an option in the current question?

Reply:
Yes that's right.